autoload/neomake/makers/proselint.vim

```diff
@@ -0,0 +1,7 @@
+" vim: ts=4 sw=4 et
+
+function! neomake#makers#proselint#proselint() abort
```

Comment:
Let's move it to `neomake#makers#text#proselint`: it is not a project maker after all (which means that it would run on the project, e.g. all files in this case).